### ISSUE TITLE
v2.3新版本训练

### ISSUE THREAD

**Issue by lin-0015** [PaddlePaddle/PaddleOCR#3979]
请问一下
配置文件:
Global:
  use_gpu: true
  epoch_num: 1000
  log_smooth_window: 20
  print_batch_step: 2
  save_model_dir: /paddle/PaddleOCR_v2.3/output/my_sast_r50_icdar/
  save_epoch_step: 200
  eval_batch_step: [100, 100]
  cal_metric_during_train: False
  pretrained_model: /paddle/PaddleOCR_v2.3/det_r50_vd_sast_icdar15_v2.0_train/best_accuracy
  checkpoints:
  save_inference_dir: /paddle/PaddleOCR_v2.3/inference/my_sast_r50_icdar/
  use_visualdl: True
  infer_img: /paddle/PaddleOCR_v2.3/train_data/test/
  save_res_path: /paddle/PaddleOCR_v2.3/output/my_sast_r50_icdar/predicts_sast.txt
  test_path: /paddle/PaddleOCR_v2.3/train_data/
  character_dict_path: ppocr/utils/number_dict.txt
  character_type: ch
  distributed: True

Architecture:
  model_type: det
  algorithm: SAST
  Transform:
  Backbone:
    name: ResNet_SAST
    layers: 50
  Neck:
    name: SASTFPN
    with_cab: True
  Head:
    name: SASTHead

Loss:
  name: SASTLoss
  
Optimizer:
  name: Adam
  beta1: 0.9
  beta2: 0.999
  lr:
    learning_rate: 0.001
    warmup_epoch: 5
  regularizer:
    name: 'L2'
    factor: 0.00005

PostProcess:
  name: SASTPostProcess
  score_thresh: 0.5
  sample_pts_num: 2
  nms_thresh: 0.2
  expand_scale: 1.2
  shrink_ratio_of_width: 0.3

Metric:
  name: DetMetric
  main_indicator: hmean

Train:
  dataset:
    name: SimpleDataSet
    data_dir: /paddle/PaddleOCR_v2.3/train_data/
    label_file_list: [/paddle/PaddleOCR_v2.3/train_data/Label.txt]
    transforms:
      - DecodeImage: # load image
          img_mode: BGR
          channel_first: False
      - DetLabelEncode: # Class handling label
      - SASTProcessTrain:
          image_shape: [512, 512]
          min_crop_side_ratio: 0.3
          min_crop_size: 24
          min_text_size: 4
          max_text_size: 512
      - KeepKeys:
          keep_keys: ['image', 'score_map', 'border_map', 'training_mask', 'tvo_map', 'tco_map'] # dataloader will return list in this order
  loader:
    shuffle: True
    drop_last: False
    batch_size_per_card: 4
    num_workers: 4

Eval:
  dataset:
    name: SimpleDataSet
    data_dir: /paddle/PaddleOCR_v2.3/train_data/
    label_file_list:
      - /paddle/PaddleOCR_v2.3/train_data/eval_label.txt
    transforms:
      - DecodeImage: # load image
          img_mode: BGR
          channel_first: False
      - DetLabelEncode: # Class handling label
      - DetResizeForTest:
          resize_long: 1536
      - NormalizeImage:
          scale: 1./255.
          mean: [0.485, 0.456, 0.406]
          std: [0.229, 0.224, 0.225]
          order: 'hwc'
      - ToCHWImage:
      - KeepKeys:
          keep_keys: ['image', 'shape', 'polys', 'ignore_tags']
  loader:
    shuffle: False
    drop_last: False
    batch_size_per_card: 1 # must be 1
    num_workers: 2
预训练模型下载的det_r50_vd_sast_icdar15_v2.0_train
检测训练评估过程很慢很慢,差不多能有17分钟的样子
![image](https://user-images.githubusercontent.com/65577659/132485654-77c8598c-fbc7-4ea0-afbf-98e99096d75c.png)
![image](https://user-images.githubusercontent.com/65577659/132488217-95bad545-8ae9-4f86-a9d5-16cc0164ede8.png)
内存情况:
![image](https://user-images.githubusercontent.com/65577659/132488271-6c1565b9-b7c0-439b-a995-84c68983c8e5.png)

请问一下是由于什么原因造成的呢?

**Comment by LDOUBLEV** [comment]
猜测主要是后处理比较慢，可以设置下参数：https://github.com/PaddlePaddle/PaddleOCR/blob/2dcc6a99fe5d6e0c66a4508d9413c2a864390a64/configs/det/det_mv3_db.yml#L9

这个参数表示从第0次迭代开始，每2000次迭代评估一次，你可以把第一个值设置大一些，等网络收敛了之后再执行eval

**Comment by lin-0015** [comment]
> 猜测主要是后处理比较慢，可以设置下参数：
> 
> https://github.com/PaddlePaddle/PaddleOCR/blob/2dcc6a99fe5d6e0c66a4508d9413c2a864390a64/configs/det/det_mv3_db.yml#L9
> 
> 这个参数表示从第0次迭代开始，每2000次迭代评估一次，你可以把第一个值设置大一些，等网络收敛了之后再执行eval

好的,谢谢啦~
